helm/README.md

```diff
@@ -2,20 +2,31 @@
 
 1. Create the cluster with [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
 
-   > Make sure ports 30001 and 30002 are free on your machine
+   > Make sure ports 30001, 30002 and 30003 are free on your machine
 
    ```bash
    kind create cluster --image kindest/node:v1.24.7 --config kind.yml
    ```
 
 2. (Optional) Build oncall image locally and load it to kind cluster
 
-3. ```bash
-   docker build ../engine -t oncall/engine:latest --target dev
-   kind load docker-image oncall/engine:latest
+   ```bash
+      docker build ../engine -t oncall/engine:latest --target dev
+      kind load docker-image oncall/engine:latest
+   ```
+
+   Also make sure to add the following lines to your `simple.yml` (you may also need to enable `devMode`):
+
+   ```yaml
+   image:
+   repository: oncall/engine
+   tag: latest
+   pullPolicy: IfNotPresent
+   oncall:
+   devMode: true
    ```
```

Comment:
could also consider adding a new `.yaml` file with these values + document passing in as another `--values` flag down on [line 34](https://github.com/grafana/oncall/pull/3204/files#diff-ba66b531f5db27e2cb364be1b4cda073681f7447f088ac98139ca96c1609dc16R32-R35)